plugin/vim-disapprove-deep-indentation.vim

```diff
@@ -51,8 +51,9 @@ function! g:DisapproveDeepIndent()
     endif
 
     set conceallevel=1 concealcursor=nvic
-    hi conceal ctermfg=red ctermbg=NONE guifg=red guibg=NONE
-
+    if !exists('g:LookOfDisapprovalNoStyle') || g:LookOfDisapprovalNoStyle == 0
```

Comment:
Thoughts on changing this to `if !exists('g:LookOfDisapprovalNoStyle')` so the setting of the variable's value outside the script is respected although it will need to be set before the plugin loads in .vimrc

Reply:
The initialization of this plugin is done in the `g:DisapproveDeepIndent` function which is called on BufEnter,BufNewFile,BufReadPost events. Due to this, it should be possible to define `g:LookOfDisapprovalNoStyle` anywhere in the `.vimrc` file.

Reply:
When I tested it, it goes into the if block even when I have set the value of `g:LookOfDisapprovalNoStyle` to 0.

Have you tried testing it?

Reply:
This is intentional. I had the following in mind:
- When an user does not specify `g:LookOfDisapprovalNoStyle` at all, the current styles are applied.
- When the user specifies `g:LookOfDisapprovalNoStyle=1` in the `.vimrc` file, styling is disabled.
- When an user does not want to remove the configuration completely from the `.vimrc` (or want to set the parameter programmatically for some reason), it is possible to set it to 0 by `g:LookOfDisapprovalNoStyle=0`, which has similar effect when the parameter is not set.

The intended semantics of the parameter is to "disable styling". If it's not set, or explicitly not activated (by specifying `0`), the old behavior applies, and the user get a red-colored eye. If the "disable styling" parameter is activated (by specifying `1`), the red-coloring is no more.